qsoconfig.py

```diff
@@ -131,7 +135,16 @@ def printv(msg, verb_tresh=1):
         cfg.args = None
 
     cfg.log_level = log_level
-
+    
+    # Load local settings
+    local_settings_fpath = os.path.join(os.getcwd(), LOCAL_SETTINGS_FILE)
+    if os.path.isfile(local_settings_fpath):
+        printv("Loading local settings from:\n{}".format(local_settings_fpath))
+        loadparams.load_parameters(local_settings_fpath, cfg, 'config')
+        if (param_fname is None
```

Comment:
As `param_fname` is passed to this function, then the one specified in `local_settings.ini` will be ignored.